docs/02-git-credentials.md

```diff
@@ -0,0 +1,35 @@
+# Git Credentials
+
+Studio Web will use the Git Credentials associated with your Stoplight account depending on the Git provider being used. That means if you connect a GitHub and a BitBucket account, and work on a Git Project which is hosted on BitBucket, it's going to know to use the BitBucket credentials. You don't need to think about it at all.
+
+Studio Desktop v1.x used to ask you to log in to your Stoplight account in order to use Git functionality for things like commits, pushing, pulling, etc. but you no longer need to bother with that. You can work with any VCS provider by adding your Git credentials for that provider.
+
+You should be asked to add credentials for a provider the first time you add that project, but if for some reason that doesn't work, or you need to change them, here's how it works.
+
+## Adding Credentials for Github
+
+1. Create a [new personal access token](https://github.com/settings/tokens/new) (more information in the [creating personal access tokens guide on Github](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).
+
+2. Choose "repo" permissions to allow enable read/writing files in your Git repository then generate the token.
+
+![image](../assets/images/git-creds-github-token.png)
+
+3. Click the Cog icon bottom left to open Project Settings, click Project Settings, and click the "+ Credential" button.
```

Comment:
Maybe reiterate the fact that the personal access token is stored locally on the user's computer, so they'll need to repeat this if they switch computers.